test/bntest.c

```diff
@@ -82,6 +82,7 @@
 #include <openssl/err.h>
 
 #include "../crypto/bn/bn_lcl.h"
+#include "../crypto/bn/bn_prime.h"
```

Comment:
Is there something wrong with them being unit tests? Also bear in mind that this was originally written a year and a half ago, when I don't believe there was any scheme for consistent testing.

Re: the issue with the `include`s, what would you otherwise suggest?